src/components/dialogs/substation-creation-dialog.js

```diff
@@ -283,7 +284,8 @@ const SubstationCreationDialog = ({
 
 SubstationCreationDialog.propTypes = {
     editData: PropTypes.object,
-    currentNodeUuid: PropTypes.string,
+    studyUuid: PropTypes.object,
+    currentNode: PropTypes.string,
```

Comment:
same here object instead of string

Reply:
replace    studyUuid: PropTypes.object, ==> studyUuid: PropTypes.string
               currentNode: PropTypes.string, ==> currentNode: PropTypes.object,